seL4-doc/src/appendices/tftpboot_example.md

```diff
@@ -0,0 +1,53 @@
+# Setting up a netboot server using TFTP - Instructions for Linux
+
+The most efficient way of booting from an image is to set up a TFTP server from which the board can read from. This removes the need to repeatedly flash an SD card with a new image, and instead the image can just be copied to the appropriate directory where it can be read and booted. The strategy used will depend on the operating system used, but this guide is for Linux.
+
+1. [Dnsmasq](https://thekelleys.org.uk/dnsmasq/doc.html) provides a network infrastructure for small networks and is designed to be lightweight. The program can be installed using apt-get
```

Comment:
apt-get is considered old fashioned these days, consider `apt` (and also on line 8). Perhaps also add "or whatever package manager your distribution provides".